samples/wifi/thread_coex/sample.yaml

```diff
@@ -10,6 +10,7 @@ tests:
       - nrf7002dk/nrf5340/cpuapp
     extra_args: CONFIG_MPSL_CX=y ipc_radio_CONFIG_MPSL_CX=y
                 CONFIG_COEX_SEP_ANTENNAS=y
+                CONFIG_OPENTHREAD=y
```

Comment:
This isn't needed, `overlay-ot.conf` is enabled for all which enables `NET_L2_OPENTHREAD` which selects `OPENTHREAD`.

@muraliThokala as the sample itself it thread co-ex, there is no point in having a separate `overlay-ot.conf`, we should merge it with `prj.conf`. WDYT?

Reply:
Removed the CONFIG from sample.yaml